src/exchange-connector/resourceLoader/fileSystemAdapter.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { readFileSync } from "fs-extra";
+
+export class FileSystemAdapter {
```

Comment:
This seems like an awful lot just to avoid writing
```
readFileSync(path, "utf8");
```

As you are taking one method that is used statically and wrapping it in a class you need to instantiate to use? I would advise against this.